sys/dev/sound/pci/maestro3.c

```diff
@@ -1160,7 +1160,7 @@ m3_intr(void *p)
 		pch = &sc->pch[i];
 		if (pch->active) {
 			pch->ptr = m3_pchan_getptr_internal(pch);
-			delta = pch->bufsize + pch->ptr - pch->prevptr;
+			delta = pch->ptr - pch->prevptr;
```

Comment:
It should be (bufsize + ptr) - prevptr, but you removed bufsize. Have you tested this code locally and noticed that bufsize calculation wasn't necessary?